ci/build_cpp.sh

```diff
@@ -70,7 +70,7 @@ then
     git config --global user.email "jliffla@sandia.gov"
     git config --global user.name "Jonathan Lifflander"
     git add "$CKPT_NAME"
-    git commit -m "Update magistrate_docs (auto-build)"
```

Comment:
Drive-by change: allow empty commits to avoid documentation workflow failing when there are no changes to docs.